javalib/src/main/scala/java/util/StringTokenizer.scala

```diff
@@ -1,103 +1,93 @@
 package java.util
 
-class StringTokenizer(string: String,
-                      private var delimiters: String,
-                      returnDelimiters: Boolean)
-    extends java.util.Enumeration[Object] {
+class StringTokenizer(
+  str: String,
```

Comment:
Variable name changes to follow Java 8 documentation are useful & not
gratuitous.